From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: William Blake Galbreath <Blake.Galbreath@GMail.com>
Date: Sat, 15 Aug 2020 11:18:27 -0500
Subject: [PATCH] Configurable permission message upgrades

This allows the configurable permission message in paper.yml to be blank and also support newlines

diff --git a/src/main/java/org/bukkit/command/Command.java b/src/main/java/org/bukkit/command/Command.java
index 1994f15831de1ca1bb7b4f52c23567825766d3f9..514efc8c79756a4ce7bc854586f9593af1d0180c 100644
--- a/src/main/java/org/bukkit/command/Command.java
+++ b/src/main/java/org/bukkit/command/Command.java
@@ -184,17 +184,11 @@ public abstract class Command {
             return true;
         }
 
-        if (permissionMessage == null) {
-            // Paper start
-            String bukkitPermissionMessage = Bukkit.getPermissionMessage();
-            if (org.apache.commons.lang.StringUtils.isNotBlank(bukkitPermissionMessage)) {
-                target.sendMessage(bukkitPermissionMessage);
-            }
-            // Paper end
-        } else if (permissionMessage.length() != 0) {
-            for (String line : permissionMessage.replace("<permission>", permission).split("\n")) {
-                target.sendMessage(line);
-            }
+        // Purpur start
+        String msg = this.permissionMessage != null ? this.permissionMessage : Bukkit.getPermissionMessage();
+        for (String line : msg.replace("<permission>", permission).split("\n")) {
+            if (org.apache.commons.lang.StringUtils.isNotBlank(line)) target.sendMessage(line);
+            //Purpur end
         }
 
         return false;
